benchmark/benchmark_gorums.pb.go

```diff
@@ -170,6 +170,28 @@ func (c *Configuration) AsyncQuorumCall(ctx context.Context, in *Echo) *AsyncEch
 	return &AsyncEcho{fut}
 }
 
+// Benchmark is the client-side Configuration API for the Benchmark Service
+type BenchmarkConfigurationClient interface {
+	StartServerBenchmark(ctx context.Context, in *StartRequest) (resp *StartResponse, err error)
+	StopServerBenchmark(ctx context.Context, in *StopRequest) (resp *Result, err error)
+	StartBenchmark(ctx context.Context, in *StartRequest) (resp *StartResponse, err error)
+	StopBenchmark(ctx context.Context, in *StopRequest) (resp *MemoryStatList, err error)
+	QuorumCall(ctx context.Context, in *Echo) (resp *Echo, err error)
+	AsyncQuorumCall(ctx context.Context, in *Echo) *AsyncEcho
+	SlowServer(ctx context.Context, in *Echo) (resp *Echo, err error)
+	Multicast(ctx context.Context, in *TimedMsg, opts ...gorums.CallOption)
+}
+
+// enforce interface compliance
+var _ BenchmarkConfigurationClient = (*Configuration)(nil)
+
+// Benchmark is the client-side Node API for the Benchmark Service
+type BenchmarkNodeClient interface {
```

Comment:
We should avoid generating empty interfaces, and the corresponding interface compliance check if it is empty. See other comments for suggestions.